core/src/main/java/com/linecorp/armeria/client/HttpRequestSubscriber.java

```diff
@@ -281,6 +281,7 @@ private void write0(HttpObject o, boolean endOfStream, boolean flush) {
             final HttpData data = (HttpData) o;
             future = encoder.writeData(id, streamId(), data, endOfStream);
             logBuilder.increaseRequestLength(data.length());
+            logBuilder.writeRequestContentPreview(data);
```

Comment:
`increaseRequestLength` and `writeRequestContentPreview` could be merged into a single method?